cookbook/core/basic/hello_flytekit.py

```diff
@@ -14,7 +14,7 @@
 # def say_hello(name: str) -> str:
 @task
 def say_hello() -> str:
-    return "hello world"
+    return "Hello, Flytekit!"
```

Comment:
If we do decide to change `hello world` we should also update the corresponding script:
https://github.com/flyteorg/flytesnacks/blob/master/cookbook/core/basic/hello_world.py